README.md

```diff
@@ -20,6 +22,10 @@ Additional description about the project and its features.
 
 [Live Demo Link](https://livedemo.com)
 
+## Video Presentation
+
+[Video demo](https://www.loom.com/share/d94cb47c28e3407bb69af558e4d8fec6)
+
 
 ## Getting Started
 
```

Comment:
- [ ] [OPTIONAL] You have here an empty section. Please, try to fulfill them or remove them. It will be better if you will write down how to set up your project on the local machine (by git clone command) and etc. 👍